src/components/app/ProfileSharing.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
```

Comment:
So, the goal is to completely get rid of propTypes using flow typing instead. You can have a look at https://github.com/DanielNill/perf.html/blob/4a7590769f01a50d175d3441f32743cf5fc9f178/src/components/app/Home.js#L51-L56 for a good example. 

Here, you still kept propTypes but using flow syntax -- this doesn't work that way :)